src/components/dashboard/main-content/course-enrollments/course-cards/AssignedCourseCard.jsx

```diff
@@ -5,30 +5,19 @@ import { Button } from '@edx/paragon';
 import BaseCourseCard from './BaseCourseCard';
 
 import { COURSE_STATUSES } from '../data';
-import dayjs from '../../../../../utils/dayjs';
 
 const AssignedCourseCard = (props) => {
   const renderButtons = () => (
-    <Button className="btn-xs-block" variant="brand">
+    <Button className="btn-xs-block bg-danger text-light rounded-0" variant="outline-danger">
```

Comment:
@mahamakifdar19 I think we should `brand` here as this is a primary action button. `danger` should be used for destructive actions.